src/schema/rgdowntime.xsd

```diff
@@ -5,7 +5,7 @@
       <xsd:element minOccurs="0" maxOccurs="unbounded" name="Downtime">
         <xsd:complexType>
           <xsd:sequence>
-            <xsd:element name="ID" type="xsd:int" />
+            <xsd:element minOccurs="0" name="ID"/>
```

Comment:
If ID is set, should we still make sure that it's an int?

Reply:
No because it might be present but empty -- as is currently the case.

Reply:
Actually maybe I should remove the minOccurs="0"

Reply:
Arghhh. Does the `rg_reader` always set an ID?

Reply:
Yep. Turns out our clients don't like missing fields, even if the fields our empty.

Reply:
When you say "even if the fields are empty", do you mean that our consumers are ok with the field being empty?